Twinpack.iss

```diff
@@ -38,7 +38,7 @@ InfoBeforeFile=DISCLAIMER
 [Files]
 Source: "TwinpackVsix\bin\{#MyConfiguration}\Package\*"; DestDir: "{#TcXaeShellExtensionsFolder}Zeugwerk\Twinpack"; Flags: ignoreversion recursesubdirs createallsubdirs; Check: InstallVsixInTcXaeShell;
 Source: "TwinpackVsix\bin\{#MyConfiguration}\TwinpackVsix.vsix"; DestDir: "{tmp}"; Flags: deleteafterinstall;
-Source: "vswhere.exe"; DestDir: "{tmp}"; Flags: deleteafterinstall;
+Source: "vswhere.exe"; DestDir: "{#TcXaeShellExtensionsFolder}Zeugwerk\Twinpack"; Flags: ignoreversion recursesubdirs createallsubdirs;
```

Comment:
Why store vswhere permanently, can’t you Extract to temp like it is done on install

Reply:
ExtractTemporaryFile('vswhere.exe');

Reply:
nope, does not work because extraction is only working with initial setup file

Reply:
Then maybe change it in the install part as well to use the local copy?

Reply:
at this time vswhere is not yet extracted to installation folder, i would leave it the way it is...